src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/CngPropertyOptions.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+
+namespace System.Security.Cryptography
+{
+    /// <summary>
+    ///     Maps to the "dwFlags" parameter of NCryptGetProperty() and NCryptSetProperty().
```

Comment:
An awful lot of the enum `<summary>` values seem to be notes to a reader/editor of the code, rather than notes to the consumer of the code.